test/unit/vl-accordion.test.html

```diff
@@ -142,6 +151,12 @@
         });
         link.click();
       });
+
+      test('Titel kan ook via slot toegevoegd worden', () => {
+        const accordion = fixture('vl-accordion-fixture-titel-slot');
+        const titel = accordion.shadowRoot.querySelector('button').querySelector('slot[name="title"]').assignedElements()[0];
+        assert.equal(titel.textContent, 'Titel via slot');
```

Comment:
```suggestion
        assert.equal(accordion._titleElement.textContent, accordion.querySelector('[slot="title"]'));
```

In unit testen kan/mag je gebruik maken van de (private) getters. Hierdoor falen de unit testen niet als bijvoorbeeld de button/title selector wijzigt.